pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -206,7 +178,7 @@ func NewKubeGenericRuntimeManager(
 	internalLifecycle cm.InternalContainerLifecycle,
 	legacyLogProvider LegacyLogProvider,
 	runtimeClassManager *runtimeclass.Manager,
-	remoteRuntimeEndpoints string,
+	_runtimeRegistry runtimeregistry.Interface,
```

Comment:
any particular reason for the leading underscore?

Reply:
committed fixes.